astro/src/components/sections/BikesSection.astro

```diff
@@ -20,5 +21,21 @@ import BaseContainer from "@components/base/BaseContainer.astro";
     </h2>
 
     <SearchBikes lang="en" />
+
+    <div
+      class="mt-4 md:mt-10 grid gap-4 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4"
+    >
+      {
+        Array.from(Array(10).keys()).map(() => (
+          <BikesCard
+            date="28/05/2022"
```

Comment:
About date: the input valve is in form YYYY-mm-dd, and in output should be formatted using the proper, language based, format, and also include a relative notation ( x days ago)
Moment js is a library i know to perform these kinds of operations

Reply:
I've decided not to use moment.js as it is fairly heavy dependency. Instead I use Intl.RelativeTimeFormat.